addons/remaining-replies/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Remaining replies for studio comments",
+  "description": "Show number of remaining replies for studio comment threads.",
```

Comment:
I have a few description ideas:
- Shows you how many more times a comment thread in a studio can receive a reply.
- Shows you how many more replies can be posted on each comment thread in a studio.
- Shows you each studio comment thread's number of remaining replies.
- Shows you the number of remaining replies for each comment thread in a studio.
- Shows you how many more times you can reply to each comment thread in a studio.

Which one do you think is the best?

Reply:
> I have a few description ideas:
> 
> * Shows you how many more times a comment thread in a studio can receive a reply.
> * Shows you how many more replies can be posted on each comment thread in a studio.
> * Shows you each studio comment thread's number of remaining replies.
> * Shows you the number of remaining replies for each comment thread in a studio.
> * Shows you how many more times you can reply to each comment thread in a studio.
> 
> Which one do you think is the best?

I prefer "Shows you how many more replies can be posted on each comment thread in a studio."